test/health-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+export AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:-ap-southeast-2}
```

Comment:
I find this default error prone. Though I do not have full picture of usage of these scripts.

Reply:
This is the region where we run the tests, it's more like hardcoded but I kept the option to be overwritten externally